src/components/ModalBlockFlow.tsx

```diff
@@ -35,14 +35,17 @@ const ModalBlockFlow: FC<ModalProps> = ({ isOpen, setIsOpen }) => (
             shards related to itself: namely <code>(X,B)</code> and <code>(B,Y)</code>, where <code>X, Y ∈ G</code>
           </li>
           <li>
-            The amount of data that a single node needs to save is reduced from <code>G2</code> to <code>2G-1</code>
+            The amount of data that a single node needs to save is reduced from <code>G²</code> to <code>2G-1</code>
```

Comment:
@polarker just verifying with you that this specific fix indeed represents the correct algorithm.

Reply:
yes, that's totally right